frontend/packages/console-dynamic-plugin-sdk/src/utils/fetch/console-fetch-utils.ts

```diff
@@ -50,7 +50,8 @@ export const validateStatus = async (
   }
 
   if (response.status === 401 && shouldLogout(url)) {
-    authSvc.logout(window.location.pathname);
+    // FIXME: Remove reference to `store`
+    authSvc.logout(window.location.pathname, InternalReduxStore.getState().UI.get('activeCluster'));
```

Comment:
Do we want to try to remove the reference to the store now?  Not sure if there is an existing selector we can replace it with.